packages/wonder-blocks-search-field/src/components/search-field.stories.js

```diff
@@ -0,0 +1,34 @@
+// @flow
```

Comment:
For some reason, this is showing up under BirthdayPicker in storybook in the first (unnamed?) section. How do I change the placement? Where should it go?